src/core/ConfigManager.cpp

```diff
@@ -126,6 +126,7 @@ void ConfigManager::upgrade_1_1_90()
 void ConfigManager::upgrade_1_1_91()
 {
 	// rename displaydbv to displaydbfs
+	// Note: the value of "displaydbfs" is not evaluated anymore!
```

Comment:
Do we need this comment? I don't think the new feature invalidates the older compatibility upgrades in any way.

Reply:
Removed the comment with commit 8d5c5230a46.